### PR TITLE
Fix: #3130 - Missing deepspeech_training.util.text.Alphabet

### PR DESCRIPTION
**Pull request by tilmankamp**
X-DeepSpeech: NOBUILD

### CODE REVIEW

File: bin/import_lingua_libre.py [hunk 1]
```diff
@@ -20,7 +20,7 @@
     get_validate_label,
     print_import_report,
 )
-from deepspeech_training.util.text import Alphabet
+from ds_ctcdecoder import Alphabet
```
Comment (lissyx) [inline bin/import_lingua_libre.py:23]: Same

File: bin/import_m-ailabs.py [hunk 2]
```diff
@@ -18,7 +18,7 @@
     get_validate_label,
     print_import_report,
 )
-from deepspeech_training.util.text import Alphabet
+from ds_ctcdecoder import Alphabet
```
Comment (lissyx) [inline bin/import_m-ailabs.py:21]: Same

File: bin/import_slr57.py [hunk 3]
```diff
@@ -20,7 +17,7 @@
     get_validate_label,
     print_import_report,
 )
-from deepspeech_training.util.text import Alphabet
+from ds_ctcdecoder import Alphabet
```
Comment (lissyx) [inline bin/import_slr57.py:20]: Same

File: bin/import_swc.py [hunk 4]
```diff
@@ -24,7 +24,7 @@
 
 from deepspeech_training.util.downloader import SIMPLE_BAR, maybe_download
 from deepspeech_training.util.importers import validate_label_eng as validate_label
-from deepspeech_training.util.text import Alphabet
+from ds_ctcdecoder import Alphabet
```
Comment (lissyx) [inline bin/import_swc.py:27]: Same

File: bin/import_tuda.py [hunk 5]
```diff
@@ -16,7 +16,7 @@
 
 from deepspeech_training.util.downloader import SIMPLE_BAR, maybe_download
 from deepspeech_training.util.importers import validate_label_eng as validate_label
-from deepspeech_training.util.text import Alphabet
+from ds_ctcdecoder import Alphabet
```
Comment (lissyx) [inline bin/import_tuda.py:19]: Same

**Comment by tilmankamp** [comment]
Regarding the re-export comment: What's the advantage of getting it indirectly through `deepspeech_training.util.text`?

**Comment by reuben** [comment]
FWIW, my suggestion was simply to restore the old `util.text` API, and reduce the diff size for this fix. Otherwise I don't mind the direct imports.

**Pull request merged**